packages/salesforcedx-vscode-core/src/channels/channelService.ts

```diff
@@ -85,6 +85,10 @@ export class ChannelService {
     });
   }
 
+  public showCommandWithTimestamp(commandName: string) {
```

Comment:
The next updates will have the rest of the code in this service using this new method.